pyat/at/tracking/particles.py

```diff
@@ -5,7 +5,8 @@
 from at.physics import ohmi_envelope
 from at.lattice.constants import clight, e_mass
 from at.physics import get_mcf, get_tune
```

Comment:
Better using  `at.constants` rather that `at.lattice.constants`. Constants has been moved to the top level, they are not specifically related to lattice.